projects/igniteui-angular/src/lib/toast/toast.component.ts

```diff
@@ -300,10 +345,9 @@ export class IgxToastComponent implements IToggleView, OnInit, OnDestroy {
      */
     public hide(): void {
         this.onHiding.emit(this);
-        this.isVisible = false;
+        this._isVisible = false;
```

Comment:
same as above, it is better to set this in `onClosed` handler of `IgxOverlayService`.

Reply:
Same here.